### ISSUE TITLE
Day of launch: Quickstarts v2 (Nov 1, 2023)

### ISSUE THREAD

**Issue by runleonarun** [fishtown-analytics/docs.getdbt.com#4177]
This tracking issue outlines the day of launch activities to complete before we merge the `Quickstarts v2` megabranch into `current`, and any wrap-up work. When this issue is closed, it means we've successfully published these docs 🎉 

## Responsible parties

- [ ] @runleonarun 

## Pre-launch checks
- [ ] Compile a few smoke tests, which includes major pages to check after we merge the megabranch PR as well as redirects we want to check.
- [ ] Review all PRs being merged into the megabranch
- [ ] Add anything else we need to complete before launch.

## Morning of release
- [ ] Update the megabranch with changes from main
- [ ] Merge the megabranch :zap:

## Testing and wrap up
- [ ] Run smoke tests
- [ ] announce in internal and community Slack channels
- [ ] Close this issue out and take a break! 🎉

**Comment by runleonarun** [comment]
🚢 d
